modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,141 @@
+
+Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of the following authentication methods:
+
+* One-time password generator
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+After a user has successfully verified their identity on a trusted device using multi-factor authentication, they are not required to reverify their identity for 90 days.
+
+== Multi-factor Authentication Configurations
+
+You can configure at least one of the following verification methods to comply with multi-factor authentication requirements:
+
+One-time password generator:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID, to verify your identity when logging in to Anypoint Platform.
+
+Security key:: Registers a USB security key to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users must configure their authentication method when they log in to Anypoint Platform after you initially enable multi-factor authentication.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements.
+--
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Select *Required*.
+. If you want to exclude users, such as service accounts, from multi-factor authentication, add users to the *Exempt Accounts* section.
+. Click *Save*. +
+All users in your organization are now required to configure a multi-factor authentication method when they log into Anypoint Platform. +
+If a user already has multi-factor authentication configured for their account, they are not prompted to configure another method.
+
+=== Exempt Users from Multi-factor Authentication
+
+Accounts that make programmatic calls to Anypoint Platform, or _service users_, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password.
```

Comment:
```suggestion
Accounts that make programmatic calls to Anypoint Platform, or _service users_, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password. 

To lower the risk of compromised user accounts, use the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. 
```

Reply:
Feels repetitious. Recommendations, @jbyrum or @wpoch ?

Reply:
I don't see the repetition there, but what I do see, is that we recommend _Connected Apps_ as mechanism to avoid a security risk (compromise the account). From my POV, that's not the intent of Connected Apps, while is something that you will achieve by using Connected Apps, I think we should be very specific that Connect Apps is the recommended way to integrate with Anypoint Platform, and user accounts shouldn't be used for that.

Reply:
For more efficient integration, use the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts.

Reply:
What about this:

"Accounts that make programmatic calls to Anypoint Platform, or _service users_, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password. 

As a best practice, MuleSoft recommends using the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts.  Connected Apps provide more security and control than user accounts."

Reply:
```suggestion
Accounts that make programmatic calls to Anypoint Platform, or service users, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password.

As a best practice, MuleSoft recommends using the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. Using the Connected Apps feature provides more security and control than using user accounts.
```